File: HA9_4.java
/*Accept N numbers from user and display all such elements which are divisible by 3 and 5
Input : N= 6
Elements : 85 66 15 80 93 88
Output : 15
*/

import java.util.*;

class ArrayX
{
    public int Arr[];
    
    public ArrayX(int iSize)
    {
        Arr = new int[iSize];
    }

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);
        int iCnt = 0;
        System.out.println("Enter the Elements : ");
        
        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }
    }

    public void Display()
    {
        Scanner sobj = new Scanner(System.in);
        int iCnt = 0;
        System.out.println("Elements are : ");
        
        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            System.out.println(Arr[iCnt] + "\t");
        }
        System.out.println();
    }

    void Difference()
    {
        int iSumE = 0;
        int iSumO = 0;
        int iCnt = 0;

        for(iCnt =0; iCnt<Arr.length; iCnt++)
        {
            if(Arr[iCnt]%3 ==0 && %5 ==0)
            {
                System.out.println(Arr[iCnt]);
            }  
        }
    }
}

class HA9_4
{
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number of Elements");
        int iSize = sobj.nextInt();

        ArrayX aobj = new ArrayX(iSize);
        aobj.Accept();
        aobj.Display();
        aobj.Difference();

        sobj.close();
    }
}

File: HA9_4.ts
import * as readline from 'readline'

function calculateDifference(arr: number[]): void {
    let evenSum = 0;
    let oddSum = 0;

    arr.forEach(element => {
        if (element % 5 === 0 && element %3 ===0) {
            console.log(element)
        } 
    });
}

function askQuestion(rl: readline.Interface, question: string): Promise<string> {
    return new Promise(resolve => {
        rl.question(question, (answer: string) => {
            resolve(answer);
        });
    });
}

async function main() {
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });

    let size: number = parseInt(await askQuestion(rl, "Enter number of elements: ")) || 0;

    askQuestion(rl, `Enter ${size} elements separated by space: `)
        .then(input => {
            const arr = input.split(' ').map(Number);
            console.log("Elements of array are:");
            arr.forEach(element => {
                console.log(element);
            });

            const difference = calculateDifference(arr);
            console.log(`Difference between summation of even and odd elements is: ${difference}`);

            rl.close();
        })
        .catch(error => {
            console.error("An error occurred:", error);
            rl.close();
        });
}

main();